src/UI/SignTransactionsModals/components/TxStatusToast.tsx

```diff
@@ -0,0 +1,34 @@
+import { TransactionStatusToast } from '../../../components/TransactionStatusToast/TransactionStatusToast';
+import React from 'react';
+
+export type TxStatusToastProps = {
+  signError: string | null;
+  canceledTransactionsMessage: string | null;
+  onDelete: () => void;
+};
+
+export const TxStatusToast: React.FC<TxStatusToastProps> = ({
+  signError,
+  canceledTransactionsMessage,
+  onDelete
+}) => {
+  if (signError) {
+    return (
+      <TransactionStatusToast
+        message={signError}
```

Comment:
We can render just one instance of TransactionStatusToast and just change the props

Reply:
I'd keep the logic at high level. The same component is used, but conditioned by `signError` and `canceledTransactionsMessage`. Why should I use a single instance and conditionally pass the props?